src/components/backend-ai-resource-monitor.ts

```diff
@@ -301,7 +301,6 @@ export default class BackendAiResourceMonitor extends BackendAIPage {
         }
 
         .resources .monitor {
-          margin-right: 20px;
```

Comment:
This code is not related to the border that you mentioned in the description.
It's related to below:
<img width="433" alt="image" src="https://github.com/lablup/backend.ai-webui/assets/621215/9917a1dc-b940-43ef-94ff-be40e3387e60">

I'll revert this part and merge this PR. If you can reproduce the second bug, please create another PR